src/gobexport/exporter/config/brk2/kadastraleobjecten.py

```diff
@@ -148,6 +453,249 @@ class KadastraleobjectenExportConfig:
     }
   }
 }
+"""
+
+    esri_format_no_subjects = KadastraleobjectenEsriNoSubjectsFormat()
+    esri_query = """
+{
+  brk2Kadastraleobjecten(indexletter:"G") {
+    edges {
+      node {
+        identificatie
+        volgnummer
+        aangeduidDoorGemeente
+        aangeduidDoorKadastralegemeentecode
+        aangeduidDoorKadastralegemeente
+        aangeduidDoorKadastralesectie
+        perceelnummer
+        indexletter
+        indexnummer
+        soortGrootte
+        grootte
+        isOntstaanUitGPerceel {
+          edges {
+            node {
+              identificatie
+            }
+          }
+        }
+        koopsom
+        koopsomValutacode
+        koopjaar
+        indicatieMeerObjecten
+        soortCultuurOnbebouwd
+        soortCultuurBebouwd
+        status
+        toestandsdatum
+        indicatieVoorlopigeKadastraleGrens
+        inOnderzoek
+        geometrie
+        invRustOpKadastraalobjectBrkZakelijkerechten(akrAardZakelijkRecht:"VE") {
+          edges {
+            node {
+              identificatie
+              aardZakelijkRecht
+              betrokkenBijAppartementsrechtsplitsingVve {
+                edges {
+                  node {
+                    identificatie
+                    statutaireNaam
+                    typeSubject
+                    heeftRsinVoor
+                    heeftKvknummerVoor
+                    heeftBsnVoor
+                    rechtsvorm
+                    statutaireNaam
+                    statutaireZetel
+                  }
+                }
+              }
+              invVanZakelijkrechtBrkTenaamstellingen {
+                edges {
+                  node {
+                    aandeel
+                    vanKadastraalsubject {
```

Comment:
Shouldn't this be vanBrkKadastraalsubject?

Reply:
This is the BRK1 tenaamstellingen `vanKadastraalsubject`.

Reply:
That means there would be a relation between BRK to BRK2, which shouldn't be the case..

Reply:
I'll rework the export now that the other BRK2 collections are available.